.gitignore

```diff
@@ -1,4 +1,5 @@
 .pioenvs
 .clang_complete
 .gcc-flags.json
-*~
+.piolibdeps
+*~
```

Comment:
Nothing, just the was the diff is displayed, maybe I added a newline at the end, the line is still there.